bin/docker_run

```diff
@@ -18,6 +18,28 @@ bundle exec rails db:seed
 
 rm -f /circuitverse/tmp/pids/server.pid
 
-echo "Starting on 127.0.0.1:3000"
+echo "Creating symlink"
+if [ ! -d "/home" ]; then
+  mkdir /home
+  echo "Created /home"
+fi
+
+cd /home
+
+if [ ! -d "${HOST_CURRENT_DIRECTORY%/*}" ]; then
+  mkdir -p "${HOST_CURRENT_DIRECTORY%/*}"
+  echo "Created ${HOST_CURRENT_DIRECTORY%/*}"
+fi
+
+if [ ! -L "$HOST_CURRENT_DIRECTORY" ]; then
+  ln -s -T /circuitverse "$HOST_CURRENT_DIRECTORY"
+  echo "Created symlink."
+else
+  echo "Symlink already exists."
+fi
+
+cd /circuitverse
+echo "Starting webserver at 127.0.0.1:3000"
 echo "Remote debugging on 127.0.0.1:3001"
-bundle exec rdbg  --nonstop --open --host 0.0.0.0 --port 3001 -c -- bundle exec rails s -p 3000 -b '0.0.0.0'
\ No newline at end of file
+echo "Starting solargraph server at 127.0.0.1:3002"
+bundle exec rdbg  --nonstop --open --host 0.0.0.0 --port 3001 -c -- bundle exec rails s -p 3000 -b '0.0.0.0'  & solargraph socket --host=0.0.0.0 --port=3002
```

Comment:
Is it possible to run this on a different container?

Reply:
@tachyons moved solargraph to seperate container